server/src/middlewares/error.middleware.ts

```diff
@@ -0,0 +1,20 @@
+import { Request, Response, NextFunction } from 'express';
+import Logger from '../lib/logger';
+
+export const errorHandler = (
+  err: Error & { statusCode?: number },
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  const statusCode = err.statusCode || 500;
+  Logger.error({
```

Comment:
should it be a middleware .. or a utility for global error handler ??

Reply:
lets use it as a middleware for now

will discuss and figure it out later 
when start working on issue #10